### ISSUE TITLE
Bot does not exit on 'exit' signal

### ISSUE THREAD

**Issue by mudol** [freqtrade/freqtrade#10421]
## Describe your environment

  * Operating system: Ubuntu 22.04
  * Python Version: 3.10.12 (`python -V`)
  * CCXT version: 4.3.54 (`pip freeze | grep ccxt`)
  * Freqtrade Version: 2024.6 (`freqtrade -V` or `docker compose run --rm freqtrade -V` for Freqtrade running in docker)
  
## Your question

*Ask the question you have not been able to find an answer in the [Documentation](https://www.freqtrade.io/en/latest/)*

Hello, I am Korean and do not speak English, so I am leaving my question through Google Translator.
I am running a strategy with Binance Futures.
Entry and exit are set at market prices, and trading-stop are applied.
When an entry signal occurs, order execution may be missed, but execution is taking place.
In a profitable state, the trade is terminated when the liquidation signal occurs, but in a loss state, the order is not executed at all at the liquidation signal, and only ends with a trading-stop and stop loss.
What went wrong?

**config.json**
---
```python
{
    "max_open_trades": 10,
    "stake_currency": "USDT",
    "stake_amount": "unlimited",
    "tradable_balance_ratio": 0.99,
    "fiat_display_currency": "USD",
    "dry_run": false,
    "dry_run_wallet": 1000,
    "cancel_open_orders_on_exit": false,
    "trading_mode": "futures",
    "margin_mode": "isolated",
    "unfilledtimeout": {
        "entry": 5,
        "exit": 5,
        "exit_timeout_count": 0,
        "unit": "minutes"
    },
    "order_types": {
        "entry": "market",
        "exit": "market",
        "emergency_exit": "market",
        "force_exit": "market",
        "force_entry": "market",
        "stoploss": "market",
        "stoploss_on_exchange": false,
        "stoploss_on_exchange_interval": 60
    },
    "entry_pricing": {
        "price_side": "other",
        "use_order_book": true,
        "order_book_top": 1,
        "price_last_balance": 0.0,
        "check_depth_of_market": {
            "enabled": false,
            "bids_to_ask_delta": 1
        }
    },
    "exit_pricing":{
        "price_side": "other",
        "use_order_book": true,
        "order_book_top": 1
    },
    "pairlists": [
        {
            "method": "StaticPairList",
            "number_assets": 21,
            "sort_key": "quoteVolume",
            "min_value": 0,
            "refresh_period": 1800
        },
        {"method": "AgeFilter", "min_days_listed": 10},
        {"method": "SpreadFilter", "max_spread_ratio": 0.005},
        {
            "method": "VolatilityFilter",
            "lookback_days": 10,
            "min_volatility": 0.05,
            "max_volatility": 0.50,
            "refresh_period": 86400
        }
    ],
    "edge": {
        "enabled": false,
        "process_throttle_secs": 3600,
        "calculate_since_number_of_days": 7,
        "allowed_risk": 0.01,
        "stoploss_range_min": -0.01,
        "stoploss_range_max": -0.1,
        "stoploss_range_step": -0.01,
        "minimum_winrate": 0.60,
        "minimum_expectancy": 0.20,
        "min_trade_number": 10,
        "max_trade_duration_minute": 1440,
        "remove_pumps": false
    },
}
```

strategies.py
---
```python
import freqtrade.vendor.qtpylib.indicators as qtpylib
import numpy as np
# --------------------------------
import talib.abstract as ta
from pandas import DataFrame
import pandas as pd # noqa

from freqtrade.strategy import (BooleanParameter, CategoricalParameter, DecimalParameter, IStrategy, IntParameter, informative, stoploss_from_absolute, stoploss_from_open)

from technical.indicators import ichimoku

def bollinger_bands(stock_price, window_size, num_of_std):
	rolling_mean = stock_price.rolling(window=window_size).mean()
	rolling_std = stock_price.rolling(window=window_size).std()
	lower_band = rolling_mean - (rolling_std * num_of_std)
	return np.nan_to_num(rolling_mean), np.nan_to_num(lower_band)

from datetime import datetime # noqa
from typing import Optional # noqa

import pandas_ta as pta

class strategies(IStrategy):

	stoploss = -0.03

	timeframe = '5m'

	can_short: bool = True

	use_exit_signal = True
	exit_profit_only = True
	ignore_roi_if_entry_signal = False

	trailing_stop = True
	#trailing_only_offset_is_reached = True
	trailing_stop_positive = 0.01
	trailing_stop_positive_offset = 0.03


	def populate_entry_trend(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
		dataframe.loc[
			(
				(qtpylib.crossed_above(dataframe['close'], dataframe['ma']))
			),
			'enter_long'] = 1

		dataframe.loc[
			(
				(qtpylib.crossed_below(dataframe['close'], dataframe['ma']))
			),
			'enter_short'] = 1

	def populate_exit_trend(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
		dataframe.loc[
			(
				(qtpylib.crossed_below(dataframe['close'], dataframe['ma']))
			),
			'exit_long'] = 1

		dataframe.loc[
			(
				(qtpylib.crossed_above(dataframe['close'], dataframe['ma']))
			),
			'exit_short'] = 1
```

**Comment by xmatthias** [comment]
> Hello, I am Korean and do not speak English, so I am leaving my question through Google Translator.

:+1: i'll try to keep my answer simple to help translators (hopfully).

---


You've got `exit_profit_only = True` - which means all exit signals are ignored (not acted on) unless `profit > 0.0`.

From the documentation for `exit_profit_only`:
> Wait until the bot reaches `exit_profit_offset` before taking an exit decision.

The behavior you describe is aligned with the expectation for this setting - and based on the settings - the bot is behaving as configured.

**Comment by mudol** [comment]
> > Hello, I am Korean and do not speak English, so I am leaving my question through Google Translator.
> 
> 👍 i'll try to keep my answer simple to help translators (hopfully).
> 
> You've got `exit_profit_only = True` - which means all exit signals are ignored (not acted on) unless `profit > 0.0`.
> 
> From the documentation for `exit_profit_only`:
> 
> > Wait until the bot reaches `exit_profit_offset` before taking an exit decision.
> 
> The behavior you describe is aligned with the expectation for this setting - and based on the settings - the bot is behaving as configured.

The issue has been resolved, thank you.
